### PR TITLE
Addition of my_enumerable.rb file and MyList class (my_list.rb)

### PR DESCRIPTION
**Pull request by AKeeganDev**
This pull request features the following:

-2 Ruby files named my_enumerable.rb & my_list.rb
-my_enumerable.rb is a module
-my_list.rb imports my_enumerable into the MyList class. If running commands in IRB, the MyList object can call the methods from the MyEnumerable module to produce results.

### CODE REVIEW

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  end
+
+  def to_s
+    @list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
```
Comment (RolandM99) [inline my_list.rb:16]: - [ ] I'll suggest you to implement your `each` method as below. In MyList implement a method #each that yields successive members of @list and uses the MyEnumerable module.
the code should resemble this:
```
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```

File: my_list.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*numbers)
+    @list = numbers
```
Comment (RolandM99) [inline my_list.rb:6]: - [ ] I think the initialize method should look like this:
```
def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```
For now when we run the code, we have this error:
image
Kindly, make sure that you initialize your MyList Class.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
## Hello Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: my_list.rb (hunk 2 above)
Comment (RolandM99) [reply my_list.rb:6]: - [ ] Hello @kemigabocatherine , I did an error while running your project for the first time, It seems that it was an error from my local PC. Everything works as expected, Kindly, submit your PR for re-review, and then the next reviewer could approve your PR.
image

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @AKeeganDev and @kemigabocatherine     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**